src/hotspot/share/gc/shenandoah/shenandoahAgeCensus.hpp

```diff
@@ -107,12 +107,6 @@ class ShenandoahAgeCensus: public CHeapObj<mtGC> {
   uint *_tenuring_threshold;         // An array of the last N tenuring threshold values we
                                      // computed.
 
-  // A private work method invoked by the public compute_tenuring_threshold() method.
-  // This uses the data in the ShenandoahAgeCensus object's _global_age_table and the
-  // current _epoch to compute a new tenuring threshold, which will be remembered
-  // until the next invocation of compute_tenuring_threshold.
```

Comment:
Could you please move this documentation to `compute_tenuring_threshold()` further below. Thanks!

Reply:
Done.